src/main/java/com/aws/greengrass/componentmanager/plugins/DockerManagerService.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.componentmanager.plugins;
+
+import com.aws.greengrass.config.Topics;
+import com.aws.greengrass.dependency.ImplementsService;
+import com.aws.greengrass.lifecyclemanager.GreengrassService;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+/**
+ * Loads docker image artifact downloader. Currently just a placeholder since the downloader is part of the
+ * nucleus but when the downloader needs to be moved out into its own plugin, this service will be
+ * instantiated for the plugin.
+ */
+@ImplementsService(name = "aws.greengrass.DockerContainerManager")
```

Comment:
nit: DockerImageManager?

Reply:
Also use a constant for the service name?

Reply:
Changed to constant, not using Image in the name since I don't want to tie the component to images, so keeping it generic